### DISCUSSION TITLE
error during migration

### DISCUSSION THREAD

**Discussion by oleksiivinogradov** [WalletConnect/walletconnect-monorepo#2587] [Developer Support]
I have to use vanila JS for flutter and Unity. This is a code:

 <script type="module">
        import { EthereumClient, w3mConnectors, w3mProvider } from 'https://cdn.jsdelivr.net/npm/@web3modal/ethereum@2.4.5/+esm'
        import { Web3Modal } from 'https://cdn.jsdelivr.net/npm/@web3modal/html@2.4.5/+esm'
        import { configureChains, createConfig } from 'https://cdn.jsdelivr.net/npm/@wagmi/core@1.1.1/+esm'
        import { arbitrum, mainnet, polygon } from 'https://cdn.jsdelivr.net/npm/@wagmi/chains@1.1.0/+esm'
        window.connectWeb3modal  = async () =>  {

          // export async function connectWeb3modal() {
          console.log('connectWeb3modal');
          const projectId = '';
          const chains = [arbitrum, mainnet, polygon]

          const { publicClient } = configureChains(chains, [w3mProvider({ projectId })])
          const wagmiConfig = createConfig({
            autoConnect: true,
            connectors: w3mConnectors({ projectId, version: 1, chains }),
            publicClient
          })
          const ethereumClient = new EthereumClient(wagmiConfig, chains)
          const web3modal = new Web3Modal({ projectId }, ethereumClient)
          await web3modal.openModal()
        }
      </script>

connectWeb3
marketplace:132 connectWeb3modal

+esm:7 SyntaxError: The requested module '/npm/@walletconnect/environment@1.0.1/+esm' does not provide an export named 'isNode' (at index.js:1:1)

**Comment by finessevanes** [comment]
Hey can you include what WalletConnect package and version you're using?

**Comment by oleksiivinogradov** [comment]
> jsdelivr

jsdelivr includes all versions
